### PR TITLE
Add currency input component

### PR DESCRIPTION
**Pull request by whatterz**
Added:
- Styles
- Macro

Documentation to be added in a future PR

### CODE REVIEW

File: app/views/components/currency-input/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        }
+      }) }}
+
+      {{ mojCurrencyInput({
+        id: "amount",
+        classes: "govuk-input--width-10",
+        name: "amount",
+        currencyLabel: {
+          text: "&yen;",
+          label: "Yen",
```
Comment (adamsilver) [inline app/views/components/currency-input/index.html]: I can't see where this is used in the codebase.

File: app/views/components/currency-input/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          text: "Amount",
+          classes: 'govuk-!-font-weight-bold'
+        },
+        hint: {
+          text: "Enter the amount you want to exchange"
+        }
+      }) }}
+
+      {{ mojCurrencyInput({
+        id: "amount",
```
Comment (adamsilver) [inline app/views/components/currency-input/index.html]: this id conflicts with the input above and causes the label to break.

File: app/views/components/currency-input/index.html (hunk 1 above)
Comment (whatterz) [reply app/views/components/currency-input/index.html]: I have removed the `label` param

File: app/views/components/currency-input/index.html (hunk 2 above)
Comment (whatterz) [reply app/views/components/currency-input/index.html]: I have made the `id`s unique

**Pull request merged**